installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_aws_rules.json

```diff
@@ -183,5 +183,6 @@
 	"aws_iamuser_shouldnothave_corenetwork_privileges",
 	"deny_public_access_to_rdp_Port_3389",
 	"enable_aws_config_service",
-	"aws_enable_encryption_for_rds_instance"
+	"aws_enable_encryption_for_rds_instance",
+	"deny_public_access_to_s3_bucket_used_to_store_cloudtrail_logs"
```

Comment:
Please see comment about naming in #1869 and apply changes here as well

Reply:
Addressed in commit